doc/interop-test-descriptions.md

```diff
@@ -89,11 +89,10 @@ Client asserts:
 
 ### client_compressed_unary
 
-This test verifies the client can compress unary messages. It sends an initial
-inconsistent request to verify whether the server supports the
-[CompressedRequest][] feature. If it does, it should catch the inconsistency and
-fail the call with an `INVALID_ARGUMENT` status. If the feature is supported, it
-proceeds with two unary calls, for compressed and uncompressed payloads.
+This test verifies the client can compress unary messages by sending two unary
+calls, for compressed and uncompressed payloads. It sends an initial probing
+request to verify whether the server supports the [CompressedRequest][] feature.
+If it doesn't, the call is expected to fail with an `INVALID_ARGUMENT` status.
```

Comment:
... to verify that the server supports the CompressedRequest feature by checking the probing call fails with an INVALID_ARGUMENT status.